components/sections/ImpactBanner/ImpactBanner.scss

```diff
@@ -6,10 +6,21 @@
   height: 600px;
   display: flex;
   align-items: center;
+  padding: 0 60px;
 }
 
 .title {
   color: white;
   font-family: $poppins-semibold;
-  transform: rotate(-90deg);
+  transform: rotate(180deg);
+  writing-mode: vertical-rl;
```

Comment:
Why are you using transform and writing mode at the same time?

Reply:
this is because when using the writing mode the text appears facing the other side 

only writing mode
![image](https://user-images.githubusercontent.com/11800206/85074308-b815cf00-b192-11ea-8cd3-3bd305712ac8.png)
![image](https://user-images.githubusercontent.com/11800206/85074457-f3b09900-b192-11ea-8e9c-cba70a2d32c1.png)


Ive just check that the text-orientation is not necessary. Not sure why i put it i remember that i have an issue that it appears like 

e
j
e
...

but probably i was doing something wrong